registration/forms.py

```diff
@@ -247,7 +247,6 @@ def create_user(self, application, commit=True):
         user = User(
             username=self.cleaned_data['username'],
             email=self.cleaned_data['email'],
-            is_active=False,
```

Comment:
I was thinking more of a workflow like edX' - ie leaving it to `False` initially, and switching it to `True` only when the two emails would be verified, but still log the user directly the first time.

Note that we would need to make sure that the login error message for inactive users mentions activating their emails (or contacting us).